modules/spring-data-2.2/src/main/java/org/apache/ignite/springdata22/repository/support/IgniteRepositoryFactory.java

```diff
@@ -107,10 +117,36 @@ public IgniteRepositoryFactory(String springCfgPath) {
 
         Assert.hasText(annotation.cacheName(), "Set a name of an Apache Ignite cache using @RepositoryConfig " +
             "annotation to map this repository to the underlying cache.");
+//        https://stackoverflow.com/questions/11616316/programmatically-evaluate-a-bean-expression-with-spring-expression-language
+        String cacheName = null;
+        if (!looksLikeExpression(annotation.cacheName())) {
+            cacheName = annotation.cacheName();
```

Comment:
One line if statements must be without brackets:

```java
if (!looksLikeExpression(annotation.cacheName()))
     cacheName = annotation.cacheName();
```

Reply:
fixed.